documentation/mqtt-overview.md

```diff
@@ -0,0 +1,83 @@
+
+```mermaid
+graph
+    A("[Broker] Topic: network/facility/clientA/temp") <-- "Publish: network/facility/clientA/temp" --> B(("Client A"))
+    A <-- "Subcribe: network/facility/clientA/temp" --> C(("Client B"))
+
+```
+
+In a more complex network, there is likely many Topics per Client and the Broker will be in charge of handling thousands of topics at once. Clients can also send commands to other clients on the network via Topics to trigger a real world action, such as a heater turning on or a status light changing color.
+
+## M5 Specifics
+
+### MQTT Deployment
+
+AnkerMake has deployed MQTT via [EMQX](https://www.emqx.com/en). Most likely it's [EMQX Cloud](https://www.emqx.com/en/cloud) since the servers are hosted by [AWS](https://aws.amazon.com/). We got that info via typing the following command into a terminal:
+
+```
+nslookup make-mqtt.ankermake.com
+```
+
+That returns something similar to:
+
+```
+Non-authoritative answer:
+Name:    ankermake-emqx-nlb-60dd062d3b52769a.elb.us-east-2.amazonaws.com
+Addresses:  3.22.247.85
+          3.134.119.140
+          18.189.81.10
+Aliases:  make-mqtt.ankermake.com
+```
+
+As you can see in the `Name:` field, it's an EMQX instance running on an AWS server.
+
+### Update Behavior
+
+As long as a M5 is powered on, it publishes updates to the broker servers under the `/phone/maker/YOUR_SERIAL_NUMBER_HERE/notice` topic as a client device. The update seems to be done by exception, meaning that it only updates when a value changes.
+
+The M5 also subscribes to  commands that may be sent to it from the MQTT servers for execution.
+
+Another client device (your phone or computer) subscribes to topics associated with the target printer and can publish commands for execution on the target printer.
+
+The basic relationships for MQTT communications can be illustrated as such:
+
+```mermaid
+graph
+    A("[Broker] Ankermake MQTT Servers (Job Queue and Data)") <-- "MQTT Data (TLS)" --> B(("M5 Printers"))
+    A <-- "MQTT Data (TLS)" --> C(("Client Devices"))
+    D("AnkerMake HTTPS Servers (API)") -- "MQTT Login Info" --> C
+    C -- "Authentication Token" --> D
+```
+
+We need to utilize the AnkerMake HTTPS API to pull down the MQTT login info and MQTT encryption key needed to setup a new client device connection instance. Your **Authentication Token** (auth_token) is what you need to authenticate your identity with the AnkerMake HTTPS servers. Then, we can plug in those MQTT credentials into the MQTT server to subscribe/publish to the topics in the broker.
+
+The messages are exchanged via an encrypted tunnel (standard MQTT over TLS) but part of the payload is further encrypted using standard AES-256-CBC using a MQTT only encryption key.
```

Comment:
`s/a MQTT only/an MQTT-only`

I think we can improve the explanation here. We (you and I) both know that the mqtt payloads are encrypted with AES, but this is the reader's first meeting with this concept.

Maybe we should explain that (somewhat unusually) the mqtt packets are encrypted, even though they're sent over a secure channel, that itself requires authentication.

Then we can explain how to get that authentication - i.e. over the http api.